src/sentry/deletions/defaults/group.py

```diff
@@ -140,10 +141,12 @@ def chunk(self) -> bool:
         if the deletion has completed and if it needs to be called again."""
         events = self.get_unfetched_events()
         if events:
+            # Adding this variable to see the values in stack traces
+            last_event = events[-1]
```

Comment:
This value may be useful to see in the stack trace to make sure we're iterating correctly. I may need to log it but for now this is good enough.